app/views/layouts/application.html.erb

```diff
@@ -5,11 +5,35 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body class="h-screen">
+    <header class="bg-gray-800 text-orange-50">
+        <nav class="h-16 flex justify-between items-center w-[96%] mx-auto">
+          <div>Stochastic Soccer</div>
+          <div class="">
+            <ul class="flex items-center gap-8">
+              <li class="">
+                <%= link_to "Standings", root_path, class: "hover:text-orange-300" %>
+              </li>
+              <li class="">
+                <%= link_to "Matches", matches_path, class: "hover:text-orange-300" %>
```

Comment:
FYI This link generates a 500 error.

Reply:
Weird since it works on my end and in [production](https://stochastic-soccer.onrender.com/)

Hummm... an opportunity for me to learn how to write a test using this path.

Reply:
Rails console is indeed cool:
```
irb(main):004> Rails.application.routes.named_routes.helper_names.include?("matches_path")
=> true
```

Reply:
The error must be because I don't have any data locally, or something like that.

Reply:
Yup, I needed to change L24 of `matches/index.html.erb`

```erb
                    <% if match.date.present? && match.date > Time.now %>
                        TBP
                    <% end %>
```

Reply:
Updated in #21 

Btw, you can load some data by going to `/leagues/seed`